app/src/androidTest/java/ch/epfl/sweng/swenggolf/ListOfferActivityTest.java

```diff
@@ -97,6 +108,40 @@ public void init() {
         localDb.writeCategories(Arrays.asList(Category.values()));
     }
 
+    @Test
+    public void refreshActuallyRefreshes() {
+        Offer newOffer = FilledFakeDatabase.getOffer(0);
+        onView(withId(R.id.offers_recycler_view)).check(matches(hasChildCount(2)));
+        Database.getInstance().write(Database.OFFERS_PATH, newOffer.getUuid(), newOffer);
+        onView(withId(R.id.refresh_list_offer))
+                .perform(withCustomConstraints(swipeDown(), isDisplayingAtLeast(80)));
```

Comment:
Why 80?

Reply:
Because otherwise the view is not visible at 90% and cannot be swiped.

Reply:
Ok :)